src/styles/components/_home-search-box.scss

```
@@ -1,15 +1,73 @@
 @import '../abstracts/_abstracts-dir';
 
 .home-search-box {
-  @include flex-center();
+  width: 80%;
+
+  background-color: $light;
+  box-shadow: rem(5px 10px 20px 0px) hsla(0, 0%, 0%, 0.16);
+
+  @include respond-above('md') {
+    width: 90%;
+  }
+
+  & > input {
+    width: 75%;
+    padding: rem(5px 10px 5px 10px);
+
+    color: $dove-gray;
+
+    border: none;
+    font-size: rem(14px);
+    line-height: 1.23;
+    text-align: left;
+
+    @include respond-above('sm') {
+      padding: rem(10px 20px 10px 20px);
+      font-size: rem(16px);
+    }
+
+    @include respond-above('md') {
+      padding: rem(20px 40px 20px 40px);
+      font-size: rem(20px);
+    }
+
+    @include respond-above('lg') {
+      padding: rem(35px 60px 35px 60px);
+      font-size: rem(24px);
+    }
+  }
 
   &__button {
-    @include button-darken-effect($picton-blue);
+    position: absolute;
+    top: 10%;
+    right: 15%;
+
+    width: 5%;
+    height: 80%;
+    padding: 0%;
+
+    background-color: $light;
+    border: none;
+    outline: none;
+
+    background: {
+      image: url('../../../public/icons/search-icon.svg');
+      repeat: no-repeat;
+      size: contain;
+      position: center;
+    }
+
+    @include respond-above('sm') {
+      top: 10%;
+      right: 15%;
+    }
+
+    @include respond-above('md') {
+      top: 25%;
+      right: 10%;
```

No need for `position`s here, a simple `width` will do 🥸
```suggestion
    &__button {
    width: 10%;
    padding: 0;
    background-color: $light;
    border: none;

    background: {
      image: url('/icons/search-icon.svg');
      repeat: no-repeat;
      size: contain;
      position: center;
    }
  }
```

---

I use `position: absolute` so that the button will bring to the front of the `Search box`. And if no declaration of `height` then the button will not be shown. but I try to make it claner. like this: 

```
&__button {
    position: absolute;
    top: 10%;
    right: 15%;

    width: 5%;
    height: 80%;
    padding: 0;

    background-color: transparent;
    border: none;

    background: {
      image: url('../../../public/icons/search-icon.svg');
      repeat: no-repeat;
      size: contain;
      position: center;
    }

    @include respond-above('md') {
      right: 10%;
    }
  }
```

What do you think? 🤔

---

`position` can be one of the option, but in this case it's too complicated to use `position` when a simple `flexbox` would do 😅  Also `position` is used when it's over `flexbox` limitations.

---

I got it now 👍🏻 I'll modify it a bit since the button is chubby now 🧸